### DISCUSSION TITLE
BASE_URL populated at runtime cannot be overridden by edit and restart

### DISCUSSION THREAD

**Discussion by ghost** [plausible/analytics#3603] [Self-Hosted Support]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

On initial configuration, I specified the BASE_URL for my instance as a local network IP, I then configured nginx for a reverse proxy, and realized that the initial BASE_URL is also used to generate the quick copy link for the externally accessible script head tag.

Upon realizing this I updated the env file with the correct external address, but after a restart the quick copy links are still showing the original internal IP.

I suspect this should be a pretty simple code change, but it seems like a bug to me.

### Expected behavior

The BASE_URL should be retrieved from the ENV file at start, not just initial start

### Environment

```markdown
- OS: Unraid
- Browser: Vivaldi 
- Browser Version: 6.4.3160.47 (Stable channel) (64-bit)
```

**Comment by ruslandoga** [comment]
👋 @luke-hagar-sp 

This is how containers work. All env vars provided at the start get baked into it and restarting the container with another set of env vars doesn't take any effect. You need to recreate the container.

**Reply by ruslandoga** [reply]
To be clear, this is not related to Plausible, and it already reads the env vars at the start, they are not being persisted anywhere.

**Reply by ghost** [reply]
Hey @ruslandoga,

Thanks for clarifying, I should have realized that.
